### PR TITLE
MB-3341 add Customer App Page Footer to storybook and move to components directory

### PR DESCRIPTION
**Pull request by Shkeating**
## Description

this branch
- creates a `.stories.jsx` file to add the Customer App Footer to storybook
- adds a wrapping `<Route>` tag around the `<Link>` tags in the footer to resolve an error
- moves the Footer component to the `components` directory
- updates the imports in other files that use the component to account for the move

## Reviewer Notes

I'm no pro at React Router, so if you are, I'd appreciate if you check to make sure I placed the `<Route>` tags in the correct location.

## Setup

```sh
yarn storybook
```
http://localhost:6006/?path=/story/customer-components-page-footer--basic 

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-3341?atlOrigin=eyJpIjoiMjQ5ZThkN2MxOGU4NGQwMmI5MTVkNDk3OGNmY2E2NjQiLCJwIjoiaiJ9) for this change

## Screenshots
image
image

### CODE REVIEW

**Review by cosburn — APPROVED** [review]

File: src/components/Customer/Footer/index.jsx [hunk 1]
```diff
@@ -21,16 +22,18 @@ function Footer() {
...
-                  <Link className="usa-footer__primary-link" to="/accessibility">
-                    Accessibility
-                  </Link>
-                </li>
-                <li className="mobile-lg:grid-col-6 desktop:grid-col-auto usa-footer__primary-content">
-                  <Link className="usa-footer__primary-link" to="/privacy-and-security-policy">
-                    Privacy and Security Policy
-                  </Link>
-                </li>
+                <Router>
```
Comment (YanZ777) [inline src/components/Customer/Footer/index.jsx:25]: I think you'll actually want to use mock providers inside of `*.stories.jsx`. 
An example you can look at is: `src/components/Customer/Profile/ContactInfoDisplay/ContactInfoDisplay.stories.jsx`

File: src/components/Customer/Footer/index.jsx [hunk 2]
```diff
@@ -21,16 +22,18 @@ function Footer() {
...
+                  <li className="mobile-lg:grid-col-6 desktop:grid-col-auto usa-footer__primary-content">
+                    <Link className="usa-footer__primary-link" to="/accessibility">
+                      Accessibility
+                    </Link>
+                  </li>
+                  <li className="mobile-lg:grid-col-6 desktop:grid-col-auto usa-footer__primary-content">
+                    <Link className="usa-footer__primary-link" to="/privacy-and-security-policy">
```
Comment (YanZ777) [inline src/components/Customer/Footer/index.jsx:32]: This link appears to be dead for me, I'm not sure if this is outside the scope of this task though.

File: src/components/Customer/Footer/index.jsx (hunk 1 above)
Comment (YanZ777) [reply src/components/Customer/Footer/index.jsx:25]: I think if you leave the <Router> wrapped around these links, they won't automatically redirect. That should be fixed once the <Router> gets moved.

**Review by YanZ777 — CHANGES_REQUESTED** [review]
I would take out the router inside of the `index.jsx` and use the mock providers inside of `index.stories.jsx` if that is where you were getting errors.

**Comment by Shkeating** [comment]
> I would take out the router inside of the `index.jsx` and use the mock providers inside of `index.stories.jsx` if that is where you were getting errors.

All done! Thanks for your help 🌟

**Review by YanZ777 — CHANGES_REQUESTED** [review]
I'm seeing a few lint errors when I make storybook, which causes the component to not show up in storybook.

**Comment by Shkeating** [comment]
> I'm seeing a few lint errors when I make storybook, which causes the component to not show up in storybook.

ahhhh i forgot to commit something 😅

**Review by YanZ777 — APPROVED** [review]
Thanks for making all those changes!

**Pull request merged**